core/src/main/java/org/apache/iceberg/rest/requests/UpdateNamespacePropertiesRequest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest.requests;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableSet;
+
+/**
+ * A REST request to set and/or remove properties on a namespace.
+ * <p>
+ * The request body specifies a list of properties to remove and a map
+ * of key value pairs to update.
+ * <p>
+ * Properties that are not in the request are not modified or removed by this call.
+ * Server implementations are not required to support namespace properties.
```

Comment:
This last paragraph is okay, but I would usually not include these here because they aren't relevant to correctly creating the request. The first one is debatable since it covers how to use the request, but the second is a higher level concern. I would omit these because they will be documented elsewhere and having them here means another place we need to locate and update if anything changes.

Reply:
I did pull this from the OpenAPI doc, so it is indeed in two places. Will remove.

Reply:
Removed.